server/ladders-remote.ts

```diff
@@ -81,10 +81,11 @@ export class LadderStore {
 		let problem = false;
 
 		if (error) {
+			const newRank = this.calculateElo(p1, p2, p1score, formatid);
 			if (error.message === 'stream interrupt') {
-				room.add(`||Ladder updated, but score could not be retrieved.`);
+				room.add(`||Ladder updated, your new rank is (${newRank}).`);
```

Comment:
Bit of a necropost but it feels like, since this does the same thing both times, it could be just one `room.add` call.
(use the first one, the `(${newRank})` makes the second one awkward.)